src/LogMonitor.js

```diff
@@ -160,7 +161,7 @@ export default class LogMonitor extends Component {
 
   handleToggleConsecutiveAction(id) {
     const { consecutiveToggleStartId } = this.props.monitorState;
-    if (consecutiveToggleStartId !== null) {
+    if (consecutiveToggleStartId > -1) {
```

Comment:
It works only first time, while `consecutiveToggleStartId` is `undefined` and the condition is not satisfied. Next time  `consecutiveToggleStartId` will be set to `null` and `null > -1`. So it will be always interpreted as a starting point, and there won't be possible to set another start point than `null` anymore.

Why not just to have `if (consecutiveToggleStartId)`? It will also prevent toggling the first INIT state. I tried and this fixes the issue.